<launch>
	<node pkg="vicon_mocap" type="vicon_recv_direct" name="vicon_recv_direct">
		<param name="stream_mode" value="ServerPush" type="str" />
		<param name="datastream_hostport" value="10.32.43.3:801" type="str" />
		<param name="flyer_subject_name" value="pelican1" type="str" />
		<param name="flyer_segment_name" value="pelican1" type="str" />
		<param name="get_camera_pose" value="false" type="bool" />
		<param name="update_rate" value="30" type="double" />
	</node>
</launch>